Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html

```diff
@@ -59,7 +59,9 @@ <h4>Check factorial for each number :</h4>
                 for (let i = 1; i < n; i++) f = f * i;
                 return f;
             }
-            document.write(`Factorial in descending Order :${factorial}`);
+            // Check Factorial in asending order
+            // document.write(`Factorial in Asending Order :${factorial}<br>`); 
+            document.write(`Factorial in descending Order :${factorial.reverse()}`);
```

Comment:
omkar reverse will only reverse the array elements...u need to sort it first n then reverse it..
https://www.w3schools.com/jsref/jsref_sort.asp